### DISCUSSION TITLE
How to set initial conditions when solving staedy problem

### DISCUSSION THREAD

**Discussion by HP199731** [PMEAL/OpenPNM#2404] [Q&A]
I set the initial concentrations of ions to 10 mol/m^3 in SalineWater class. When the charging conditions are applied, ions should adsorb on the charging surface and the concentrations of ions in closed system should not be 0 mol/m^3. However, the steady concentrations solved by OpenPNM is 0 mol/m^3 and potential is also 0 V. Therefore, boundary conditions (c=10 mol.m^3) must be applied in some pores to guarantee non-zero solution.
So, whether my way to setup initial concentrations is wrong?

**Comment by ma-sadeghi** [comment]
For starters, maybe share a minimum working example (MWE).
